Magento 2.3.5 Release Note May 20 hotfix update (#7259)
magento/devdocs: src/guides/v2.3/release-notes/release-notes-2-3-5-commerce.md

**Issue**: Magento throws an error on the Order Summary  section of the checkout workflow when a customer removes store credit after first selecting it as a payment method for the entire order. **Workaround**: Customers can refresh the Order page. A fix will be available in Magento 2.3.6, which is scheduled for release in Q4 2020.
<!-- MC-33431-->

> **Issue**: The storefront checkout workflow will not display any payment method that has been enabled for specific countries with the exception of the Klarna and Amazon Pay payment methods.  **Workaround**: Download and apply [Patch for specific country payment method issue](https://magento.com/tech-resources/download#download2369). A fix will be available in Magento 2.3.6, which is scheduled for release in Q4 2020.

Reviewer: ```suggestion
**Issue**: The storefront checkout workflow will only display the Klarna and Amazon Pay payment methods for some countries, although other payment methods have been enabled. **Workaround**: Download and apply [Patch for specific country payment method issue](https://magento.com/tech-resources/download#download2369). A fix will be available in Magento 2.3.6, which is scheduled for release in Q4 2020.
```

Author: changed text (with one tiny further modification)
